### PR TITLE
Vertical support

### PR DESCRIPTION
**Pull request by possan**
Modified the code and a couple of samples to support vertical pagination too

### CODE REVIEW

File: demo/ereader/vertical.html [hunk 1]
```diff
@@ -0,0 +1,1788 @@
...
+
+			if (upcoming != ereader.masterPages[i].dataset.pageIndex) {
+				el = ereader.masterPages[i].querySelector('div');
+				el.innerHTML = pages[upcoming];
+			}
+		}
+	});
+}, false);
+</script>
+<script type="text/plain" id="book">
```
Comment (cubiq) [inline demo/ereader/vertical.html:152]: just wondering why you didn't reuse the ajax call

**Comment by cubiq** [comment]
wow... what a mod ;) congrats! Gotta make same tests before merging.

File: demo/ereader/vertical.html (hunk 1 above)
Comment (possan) [inline demo/ereader/vertical.html:152]: i didnt have a working internet connection or webserver while working on it :) but you could just aswell ignore the demo

**Comment by cubiq** [comment]
Unfortunately I cannot merge the Engadget stuff, anyway terrific job, thanks!

**Comment by urbien** [comment]
built in vertical support in swipeview would be sweet!

**Pull request closed without merging**